### ISSUE TITLE
⭐ Onboard 2 New Apps

### ISSUE THREAD

**Issue by julialawrence** [ministryofjustice/analytical-platform#3300]
### User Story

As an Analytical Platform Customer I would like to onboard a new app onto the platform in order to make data available to my stakeholders and assist the AP team in validating their onboarding process/documentation.

### Value / Purpose

This will benefit me in allowing me to deploy an application into Cloud Platform utilising AP data while also allowing AP team to perform final checks before opening up the capability to wider audience

### Useful Contacts

Michael Collins, Julia Lawrence

### User Types

Analytical Platform customers

### Hypothesis

If the outcomes of the activity are positive we are ready to start onboarding again.

### Proposal

Jointly carry out the onboarding activity, capture the outcomes and keep track of the findings. In order to enable working in the activity will be managed in the #analytical-platform-new-app-experiments channel which can serve as a repository of issues until it can be absorbed into documentation.

### Additional Information

The team onboarding is called EMDS.
Due to how our sprints are set up, the collaboration window is one week commencing Feb 19th.

Second app is a new app authored by Laurence Droy who will provide details in the comment to this ticket. 

### Definition of Done

- [x] App onboarded
- [x] App 2 onboarded
- [x] Lessons learned captured
- [x] Information in the working channel rolled into the documentation
- [ ] Channel archived

**Comment by michaeljcollinsuk** [comment]
Releated ticket https://github.com/ministryofjustice/data-platform/issues/2981

**Comment by michaeljcollinsuk** [comment]
Apps were successfully onboarded. 

One major change made as part of the process, we updated the Control Panel to store the namespace of the app when registering it. This was implemented in https://github.com/ministryofjustice/analytics-platform-control-panel/pull/1257

Further minor feedback was provided:
- When creating an environment, no instructions about what to set for branch protection:
- Some users may not be able to install the Cloud Platform CLI https://mojdt.slack.com/archives/C06JENW39QD/p1708345614415199
- There were questions about the "Manage customers" page/functionality and how this works. The docs have been updated to make it clear this is only for apps that use the "email" connection for login.
- Text on the Register App form could be clearer. 
  - Update Github Repository fieldname to make it clear full URL is needed
  - Make it clear that the options for webapp datasources are limited to what the user has access to

Changes in Control Panel to address the above, and reenable the app registration button are included in https://github.com/ministryofjustice/analytics-platform-control-panel/pull/1261. This will be merged when we are ready to open the app registration process to all users.

The slack channel still needs to be archived, awaiting any response to request for further feedback before doing this.
